lib/ansiblelint/utils.py

```diff
@@ -76,8 +76,8 @@ def ansible_template(basedir, varname, templatevars, **kwargs):
     'name', 'action', 'when', 'async', 'poll', 'notify',
     'first_available_file', 'include', 'tags', 'register', 'ignore_errors',
     'delegate_to', 'local_action', 'transport', 'remote_user', 'sudo', 'sudo_user',
-    'sudo_pass', 'when', 'connection', 'environment', 'args',
-    'any_errors_fatal', 'changed_when', 'failed_when', 'always_run', 'delay', 'retries', 'until',
+    'sudo_pass', 'when', 'connection', 'environment', 'args', 'always_run',
+    'any_errors_fatal', 'changed_when', 'failed_when', 'check_mode', 'delay', 'retries', 'until',
```

Comment:
No need for this change, as this is only used for pre 2.0 code (2.0 onwards uses ModuleArgsParser so that I can rely on ansible's handling)